@xen-orchestra/web-core/lib/locales/en.json

```
@@ -669,26 +680,19 @@
   "success": "Success",
   "summary": "Summary",
   "support": "Support",
-  "suspend": "Suspend",
   "suspend-storage-repository": "Suspend storage repository",
   "switch-theme": "Switch theme",
   "system": "System",
-  "system-disks-health": "System disks health",
   "table-actions": "Table actions",
   "tags": "Tags",
   "task": "Task",
-  "task.ended": "@:ended",
-  "task.estimated-end": "Estimated end",
-  "task.progress": "Progress",
-  "task.started": "@:started",
+  "task:ended": "@:ended",
+  "task:started": "@:started",
   "tasks": "Tasks",
-  "tasks.n-subtasks": "{n} subtask | {n} subtasks",
-  "tasks.no-tasks": "No tasks",
-  "tasks.quick-view": "Tasks' quick view",
-  "tasks.quick-view.all": "All",
-  "tasks.quick-view.done": "Done",
-  "tasks.quick-view.failed": "Failed",
-  "tasks.quick-view.in-progress": "In progress",
+  "tasks:all": "@:all",
+  "tasks:done": "@:done",
+  "tasks:failed": "@:failed",
+  "tasks:quick-view": "Tasks' quick view",
```

```suggestion
  "tasks:quick-view": "Tasks overview",
```
The ' was intentional ?